frontend/keras.py

```diff
@@ -15,6 +15,11 @@ def transform_to_intermediate_format(self, input):
         model = keras.models.load_model(input)
         model_json = json.loads(model.to_json())
 
+        #? Remove InputLayer from json file
```

Comment:
Apparently tensorflow now creates an additional layer called the "InputLayer" as an entry point for the models (https://www.tensorflow.org/api_docs/python/tf/keras/layers/InputLayer). Since we can get all currently required information from the first actual layer, there is no need to extract informtion from this first layer. Therefore, the layer will be removed from the model_json object. Otherwise one has to implement specific information extraction routines and add it to the supported layers.